code/modules/surgery/organs/external/wings/functional_wings.dm

```diff
@@ -196,3 +196,9 @@
 	name = "megamoth wings"
 	desc = "Don't get murderous."
 	sprite_accessory_override = /datum/sprite_accessory/wings/megamoth
+
+//fly wings for flies
```

Comment:
Three slashes (///) for an autodoc comment. You should specifiy these are from the flight potion as well

Reply:
That'll do?